common/src/test/java/org/jboss/gm/common/ConfigurationTest.java

```diff
@@ -78,4 +80,14 @@ public void verifyDefaultState() {
         Configuration c = ConfigFactory.create(Configuration.class);
         assertNull(c.overrideTransitive());
     }
+
+    @Test
+    public void verifyDumpCurrentConfig() {
+        final Configuration c = ConfigFactory.create(Configuration.class);
+        final String currentConfig = c.dumpCurrentConfig();
+        final String[] lines = currentConfig.split(System.lineSeparator());
+        assertThat(lines).isNotEmpty();
+        final String[] keys = Arrays.stream(lines).map(line -> line.split(":")[0].trim()).toArray(String[]::new);
+        assertThat(keys).hasSize(lines.length).isSorted();
```

Comment:
This simply checks that the keys are sorted and the sizes of the keys used to test matches the number of lines that are in the original string output. I could check more specific things like the number of keys, but thought that might be subject to change.